json-lint/schemas/inventory/adservers/appnexus.json

```diff
@@ -0,0 +1,49 @@
+{
+    "$id": "#appnexus",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "Appnexus adserver",
+    "description": "https://github.com/ampproject/amphtml/blob/master/ads/appnexus.md",
+    "additionalProperties": false,
+    "required": [
+        "type"
+    ],
+    "defaultSnippets": [
+        {
+            "label": "Appnexus template",
+            "description": "contains required keys",
+            "body": {
+                "type": "appnexus"
+            }
+        }
+    ],
+    "properties": {
+        "type": {
+            "const": "appnexus"
+        },
+        "tagid": {
+            "type": [
+                "integer",
```

Comment:
I would put string only.

Reply:
Can also been a numerical ID in the examples. Added for completion.

Reply:
Ok. Can live with that 👍 😄